internal/hmyapi/apiv1/transactionpool.go

```diff
@@ -152,26 +152,6 @@ func (s *PublicTransactionPoolAPI) GetStakingTransactionByBlockHashAndIndex(ctx
 	return nil
 }
 
-// GetTransactionCount returns the number of transactions the given address has sent for the given block number
```

Comment:
we don't need GetTransactionCount and GetStakingTransactionCount for apiv1?

Reply:
Just deeted the self explanatory comment. Technically dont need to add on v1 but this pr just adds to v1 as well for consistency

Reply:
we actually do. this was why the api_test.sh was failing. Putting this back with comment.